File: sandbox/fixtures/rules/python/CGR_BUG_ERR_005_silent_swallow/src/clean_handler.py
# Clean: Proper exception handling
import logging

logger = logging.getLogger(__name__)

def handle():
    try:
        do_something()
    except Exception as e:
        logger.warning(f"Operation failed: {e}")
        handle_error(e)

def handle_error(e):
    # Recovery logic
    pass

File: sandbox/fixtures/rules/java/CGR_BUG_ERR_005_silent_swallow/src/CleanHandler.java
// Clean: Proper exception handling
import java.util.logging.Logger;

public class Handler {
    private static final Logger logger = Logger.getLogger(Handler.class.getName());

    public void handle() {
        try {
            doSomething();
        } catch (Exception e) {
            logger.log(java.util.logging.Level.WARNING, "Operation failed", e);
            handleError(e);
        }
    }

    private void doSomething() throws Exception {}

    private void handleError(Exception e) {
        // Recovery logic
    }
}
